MockKit/MockPumpManagerState.swift

```diff
@@ -46,6 +53,44 @@ public struct MockPumpManagerState {
             unfinalizedTempBasal = nil
         }
     }
+
+    var supportedBolusVolumes: [Double] {
+        switch deliverableIncrements {
+        case .omnipod:
+            // 0.05 units for volumes between 0.05-30U
+            return (1...600).map { Double($0) * 0.05 }
+        case .medtronicX22:
+            // 0.1 units for volumes between 0.1-25U
+            return (1...250).map { Double($0) * 0.1 }
+        case .medtronicX23:
+            let breakpoints = [0, 1, 10, 25]
+            let scales = [40, 20, 10]
+            let scalingGroups = zip(scales, breakpoints.adjacentPairs().map(...))
+            return scalingGroups.flatMap { (scale, range) -> [Double] in
+                let scaledRanges = (range.lowerBound * scale + 1)...(range.upperBound * scale)
+                return scaledRanges.map { Double($0) / Double(scale) }
+            }
+        }
```

Comment:
Implementations taken from DashPumpManager and MinimedPumpManager (and same for `supportedBasalRates` below)